docs/custom_order_filters.md

```diff
@@ -0,0 +1,58 @@
+# Custom Order Filters
+
+Mesh supports the creation of separate sub-networks where 0x orders that adhere to a specific schema are shared. Each sub-network is built around a custom order filter. The custom filter defines which orders are allowed to be shared within a sub-network. For example:
+
+- All orders for a specific asset pair (e.g., ETH/DAI)
```

Comment:
Non-blocking suggestion: It would probably be helpful to add explicit examples for the filters described here. I imagine that these will be used a lot.

Reply:
Seconded.